<?xml version="1.0" encoding="UTF-8"?>
<launch>
	
	<!-- Porta do Arduino -->
	<arg name="ard_port" default="0" />
	<arg name="maps_dir" default="/mnt/HD/ROS_maps/roboserv" />
	<arg name="server_address" default="http://192.168.1.2:5010"/>
	
	<!-- Inicia a comunicacao com o servidor -->
	<node name="app_node" pkg="roboserv_simulation" type="app_node.py" output="screen">
		<param name="ard_port" value="$(arg ard_port)"/>
		<param name="directory" value="$(arg maps_dir)"/>
		<param name="server_address" value="$(arg server_address)"/>
	</node>

</launch>
